.gitignore

```diff
@@ -38,6 +38,7 @@ install/kubernetes/addons/zipkin-to-stackdriver.yaml
 install/kubernetes/addons/zipkin.yaml
 install/kubernetes/istio-auth.yaml
 install/kubernetes/istio-mixer-with-health-check.yaml
+install/kubernetes/istio-mixer-validator.yaml
```

Comment:
This should probably also be added to the FILES_TO_CLEAN list in the Makefile.  Offhand it looks like install/kubernetes/istio-mixer-with-health-check.yaml  and install/kubernetes/istio-ca-with-health-check.yaml  also weren't added to the list, and it looks like install/kubernetes/helm/istio/values.yaml should be removed from FILES_TO_CLEAN because it was checked in to the repo (when I run "make clean" git noticed that the file disappeared).

Reply:
Done, thanks for pointing it out.